x-pack/plugins/canvas/canvas_plugin_src/uis/arguments/image_upload/index.js

```diff
@@ -15,7 +16,7 @@ import {
   EuiButton,
   EuiFieldText,
 } from '@elastic/eui';
-import { Loading } from '../../../../public/components/loading';
+import { Loading } from '../../../../public/components/loading/loading';
```

Comment:
We shouldn't be reaching into the directory to grab the file to bypass the container. The convention we've tried to follow in Kibana land is that the `index.js` file exposes the "public" api for a thing, anything else in that path is "private" (and limited to testing).

I understand that this is kind of a quick fix, can you at least add a TODO comment here?

Reply:
Right.  The problem is the `index` of that directory invokes `connect`, which is the source of the error.  If you know of a way to account for it, I can switch it:

https://github.com/clintandrewhall/kibana/blob/file-upload/x-pack/plugins/canvas/public/components/loading/index.js#L15

Otherwise, I'll add a TODO.